gradio/blocks.py

```diff
@@ -1265,8 +1266,14 @@ def reverse(text):
                 raise RuntimeError("Share is not supported when you are in Spaces")
             try:
                 if self.share_url is None:
-                    share_url = networking.setup_tunnel(self.server_port, None)
-                    self.share_url = share_url
+                    print(
+                        "\nSetting up a public link... we've recently upgraded the "
+                        "way public links are generated. But if you encounter any"
+                        "issues, please try with gradio <= 3.7.\n"
```

Comment:
Maybe we could add a `"and please report the issue on https://github.com/gradio-app/gradio"` ?

Reply:
Agreed, made the change.